shawmut/__init__.py

```diff
@@ -0,0 +1 @@
+from services import autooff
```

Comment:
This is no longer needed...somebody just left that in there. The whole line can be deleted.